projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -644,7 +640,7 @@ public static function block_attributes_to_shortcode_attributes( $atts, $type, $
 					}
 
 					$atts['options']     = implode( ',', $options );
-					$atts['optionsdata'] = \wp_json_encode( $options_data );
+					$atts['optionsdata'] = \wp_json_encode( $options_data, JSON_UNESCAPED_SLASHES );
```

Comment:
Tracking down where this gets used, we should probably include at least `JSON_HEX_AMP` here since it seems to make its way into `esc_html()`.
```suggestion
					$atts['optionsdata'] = \wp_json_encode( $options_data, JSON_UNESCAPED_SLASHES | JSON_HEX_AMP );
```